src/lib/core/CHIPTLV.h

```diff
@@ -1147,6 +1147,11 @@ class DLL_EXPORT TLVWriter
      */
     CHIP_ERROR PutBoolean(uint64_t tag, bool v);
 
+    /**
+     * @overload CHIP_ERROR TLVWriter::Put(uint64_t tag, bool v)
```

Comment:
Could you add a comment as to why booleans need to be treated differently than the regular template Put?

Reply:
Added a comment in the function body.